test/metabase/lib/js_test.cljs

```diff
@@ -183,21 +183,6 @@
       (is (test.js/= (clj->js snippets)
                      (lib.js/template-tags query))))))
 
-(deftest ^:parallel extract-template-tags-test
```

Comment:
`extract-template-tags` is no longer exported. There is another test that tests `with-template-tags` with `undefined` properties.